modules/hitlnext/src/views/full/app/components/HandoffList.tsx

```diff
@@ -36,6 +38,17 @@ const HandoffList: FC<Props> = ({ handoffs, loading }) => {
       resolved: item.status === 'resolved'
     }
 
+    if (filterOptions.tags.length > 0) {
+      const tagCondition = item.tags && item.tags.some(tag => filterOptions.tags.indexOf(tag) >= 0)
+
+      Object.assign(conditions, {
+        unassigned: item.agentId == null && tagCondition,
+        assignedMe: item.status === 'assigned' && item.agentId === state.currentAgent?.agentId && tagCondition,
+        assignedOther: item.status === 'assigned' && item.agentId !== state.currentAgent?.agentId && tagCondition,
+        resolved: item.status === 'resolved' && tagCondition
+      })
+    }
+
```

Comment:
This can be refactor like so:

```suggestion
    if (filterOptions.tags.length) {
      const tagCondition = item.tags?.some(tag => filterOptions.tags.indexOf(tag) >= 0)

      Object.keys(conditions).forEach(key => (conditions[key] = conditions[key] && tagCondition))
    }
```